src/go/cmd/update-collection-v3/migrations/logs/valuesv3.go

```diff
@@ -0,0 +1,17 @@
+package logs
+
+type ValuesV3 struct {
```

Comment:
It's misleading to name it "v3" because this is not really the "v3" version of values - it's something in between v2 and v3. How about something like "after" or "output"?

Reply:
See above comment about v2.